docs/_data/destination-service.json

```diff
@@ -0,0 +1,36 @@
+{
+  "propertyFields": [
+    {
+      "name": "username",
+      "type": "string",
+      "description": "clientID generated by Destination service"
+    },
+    {
+      "name": "password",
+      "type": "string",
+      "description": "client secret generated by Destination service"
+    },
+    {
+      "name": "destinationUrl",
+      "type": "string",
+      "description": "Destination service URL to call on retrieving destination"
+    },
+    {
+      "name": "destinationName",
+      "type": "string",
+      "description": "name of the chosen destination"
+    },
+    {
+      "name": "oAuthTokenUrl",
+      "type": "string",
+      "description": "OAuth token URL to call on retrieving token"
+    }
+  ],
+  "methods": [
+    {
+      "name": "getAuthentication",
+      "returnValue": "Promise<any>",
+      "description": "Get the authentication from the destination."
```

Comment:
What authentication stands for in this description?

Reply:
It can return Promise from oAuth or basic authentication or keystore object. Thats why the expected value is any.